openlibrary/templates/type/work/editions_datatable.html

```diff
@@ -40,7 +40,7 @@
 $ edition_list_secs = time() - edition_list_start
 
 <p class="small sansserif edition-adder">
-  <a href="/books/add?work=$work.key" title="$_('Add another edition of %(work)s', work=work.title)">$_("Add another edition?")</a>
+  <a href="/books/add?work=$work.key" title="$_('Add another edition of %(work)s', work=work.title)" rel="noreferrer">$_("Add another edition?")</a>
```

Comment:
I believe we want this to be `nofollow` to prevent crawlers from following the link, but support is limited. If this looks good to you, lgtm, so feel free to merge :+1: 
```suggestion
  <a href="/books/add?work=$work.key" title="$_('Add another edition of %(work)s', work=work.title)" rel="nofollow">$_("Add another edition?")</a>
```